app/helpers/application_helper.rb

```diff
@@ -108,11 +108,11 @@ def link_to_article_search
     )
   end
 
-  # TODO: add bento link, see issue #1695
   def link_to_bento_search
+    query_string = params[:q].present? ? "?utf8=✓&#{{ q: params[:q] }.to_query}" : nil
```

Comment:
Sorry I didn't catch this on the first pass.  I don't think we'll want to explicitly call out `utf8` in our URL.  That's a rails thing that gets passed on, we could either pass `params[:utf8]` (so it gets set to what rails set instead of forced) or just drop it all-together.

Reply:
Okie doke, I decided to drop it. Bento doesn't seem to mind. ¯\_(ツ)_/¯